starlark/tests/go-testcases/int.sky

```diff
@@ -149,10 +149,13 @@ assert_(-1 > -2)
 maxint64 = 9223372036854775807 # = 2^63
 minint64 = -maxint64 - 1       # = -2^64
 assert_eq(str(maxint64), "9223372036854775807")
-assert_eq(str(maxint64+1), "-9223372036854775808")
+# Overflow
+# assert_eq(str(maxint64+1), "9223372036854775808")
```

Comment:
Since those are tested for error in the new rust testcase, simply remove the line?

Reply:
I kept it to keep track of the differences beetween Go and Rust implementation.

But I can remove it if you think it's not needed.

Reply:
Keeping them SGTM since, if I remember correctly, I did that at a couple of other place.